src/integration/common/SEO.js

```diff
@@ -0,0 +1,36 @@
+const { amp, canonical } = global;
+
+export default ({ pageTitle, canonicalUrl, language }) => {
+  [amp, canonical].forEach(app => {
```

Comment:
Probably just a subjective thing, but would `page` be a more neutral conventional name for the argument, here and throughout? (does `app` make sense for AMP pages?)